docs/source/changes.rst

```diff
@@ -15,9 +15,13 @@ Improvements and Changes:
   that uses fixed random seeds instead of averaging over several experiments. This can
   be enabled with the `--use-seed` command line option when running `pytest` (gh-928).
 
+- Updated the examples README. Removed an outdated notebook. Updated remaining notebooks to use
+  ``MemoryReference``s, and fix any parts that were broken (gh-820).
```

Comment:
this is not rendering right -- maybe you cant start a line with the double backquoting